bitnami/mongodb/templates/replicaset/scripts-configmap.yaml

```diff
@@ -219,7 +219,7 @@ data:
     SLEEP_PERIOD=10
 
     {{- if and .Values.auth.enabled .Values.auth.rootPassword }}
-    usernameAndPassword="-u root -p ${MONGODB_ROOT_PASSWORD}"
+    usernameAndPassword="-u ${$MONGODB_ROOT_USER} -p ${MONGODB_ROOT_PASSWORD}"
```

Comment:
Shouldn't this be just `${MONGODB_ROOT_USER}`, without extra `$`?

Reply:
It absolutely should! I apologise for letting this slip through.

Reply:
I've implemented a fix here: https://github.com/bitnami/charts/pull/17116